save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/vuln/VulnerabilityView.kt

```diff
@@ -39,8 +40,22 @@ val vulnerabilityView: FC<VulnerabilitiesViewProps> = FC { props ->
     val (vulnerability, setVulnerability) = useState<VulnerabilityExt>()
     val (selectedMenu, setSelectedMenu) = useState(VulnerabilityTab.INFO)
     val (isTableView, setIsTableView) = useState(false)
+    val (countComments, setCountComments) = useState(0)
     val addProjectWindowOpenness = useWindowOpenness()
 
+    useRequest {
+        val newComments = post(
+            url = "$apiUrl/comments/get-all-count",
+            headers = jsonHeaders,
+            body = window.location.pathname,
+            loadingHandler = ::noopLoadingHandler,
+        ).unsafeMap {
+            it.decodeFromJsonString<Int>()
+        }
```

Comment:
Why do we need POST request here if we just get count?

Reply:
because GET doesn't support `body`, but @Cheshiriks maybe refactor location to request param?

Reply:
Comments are tied to the section in which we are located. Therefore, in the body of the request we need to pass a string indicating this section. And `GET` has no `body`.

Reply:
Ok, changed it

Reply:
Can we change it to smth like this? 
```
val newComments = get(
            url = "$apiUrl/comments/get-all-count?section=$window.location.pathname",
            headers = jsonHeaders,
            loadingHandler = ::noopLoadingHandler,
        ).unsafeMap {
            it.decodeFromJsonString<Int>()
        }
```

Reply:
Yes, agree. Changed it

Reply:
please use `params`, it's more clear way and also it use URLEncoder for params